gravitee-am-test/specs/gateway/enduser-logout.jest.spec.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fetch from "cross-fetch";
+
+global.fetch = fetch;
+
+import {jest, afterAll, beforeAll, expect} from "@jest/globals";
+import {requestAdminAccessToken} from "@management-commands/token-management-commands";
+import {createDomain, deleteDomain, patchDomain, startDomain} from "@management-commands/domain-management-commands";
+import {getAllIdps} from "@management-commands/idp-management-commands";
+import {createUser} from "@management-commands/user-management-commands";
+import {
+    createApplication, 
+    patchApplication, 
+    updateApplication
+} from "@management-commands/application-management-commands";
+import {
+	extractXsrfTokenAndActionResponse,
+    getWellKnownOpenIdConfiguration,
+    logoutUser,
+    performFormPost,
+    performGet,
+} from "@gateway-commands/oauth-oidc-commands";
+
+let domain;
+let managementApiAccessToken;
+let openIdConfiguration;
+let application;
+let user = {
+    "username": "LogoutUser",
+    "password": "Test123!",
+    "firstName": "Logout",
+    "lastName": "User",
+    "email": "logoutuser@acme.fr",
+    "preRegistration": false
+}
+
+jest.setTimeout(200000)
+
+const waitForDomainSync = () => new Promise((r) => setTimeout(r, 10000))
+
+beforeAll(async () => {
+    const adminTokenResponse = await requestAdminAccessToken();
+    managementApiAccessToken = adminTokenResponse.body.access_token;
+    expect(managementApiAccessToken).toBeDefined();
+    const createdDomain = await createDomain(managementApiAccessToken, "jest-logout", "test end-user logout");
+    expect(createdDomain).toBeDefined();
+    expect(createdDomain.id).toBeDefined();
+    domain = createdDomain;
```

Comment:
you can assign `domain` at line 59 directly as the `createDomain` variable does not do anything